src/nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
[OPTIONAL]
- [ ] I kindly suggest you complete handling the error by providing a descriptive error message for user, see the below code snippet for guideance :+1: :sparkles: 
- code snippet
```ruby
raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
```